apps/common/lib/lexical/ast/callable.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Lexical.Ast.Callable do
+  alias Lexical.Ast
+  alias Lexical.Ast.Analysis
+  alias Lexical.Document
+  alias Lexical.Document.Position
+  alias Lexical.Document.Range
+  alias Sourceror.Zipper
+
+  import Sourceror.Identifier, only: [is_call: 1]
+
+  def fetch_name_range(%Analysis{} = analysis, %Position{} = position) do
```

Comment:
WIP: A more appropriate approach here might be to accept `analysis`, `range`, and `name` and then call `Ast.traverse_in`.